packages/nuxt/src/core/app.ts

```diff
@@ -107,8 +107,10 @@ export async function resolveApp (nuxt: Nuxt, app: NuxtApp) {
   }
 
   // Resolve layouts/ from all config layers
+  const layerConfigs = nuxt.options._layers.map(layer => layer.config)
+  const reversedConfigs=layerConfigs.reverse()
```

Comment:
(.reverse mutates the array so you would need to spread it...)

Also, would you be able to match the spacing of the existing code 🙏

Reply:
Sure thing! I apologize, I've been editing it in place.